src/test/scala/beam/router/PhyssymXmlToOsmConverter.scala

```diff
@@ -30,10 +34,12 @@ object PhyssymXmlToOsmConverter extends StrictLogging {
     var canProcessLink = false
     var canProcessLinkAttributes = false
     var canProcessAttribute = false
+
     var isWayId = false
     var isHighway = false
+    var wayCounter = 1L
 
-    var wayId: String = null
+    var wayId: String = String.valueOf(wayCounter)
```

Comment:
The links and wayCounter change logic, whereas the PR is listed as enabling PBF. The two changes should probably be in separate PRs - or at least document the changes being made. That way future debuggers can trace changes easier if needed.

Reply:
@JustinPihony since was defined that two ways must be generated per link (one on each direction) the previous code was generating two ways with the same id. it is not only a "changing of behavior" - it is a bug fix. QGIS don't care about this bug, but the conveyal osm library keep only one (since its use a map on its data structure)
Since this code is relatively new and it was merged last week I believe this bug fix won't affect any client

Reply:
OK, can you at least add a comment to the PR description then?

Reply:
sure